app/src/main/java/de/westnordost/streetcomplete/overlays/surface/PathSurfaceOverlayForm.kt

```diff
@@ -0,0 +1,448 @@
+import de.westnordost.streetcomplete.osm.surface.SingleSurface
+import de.westnordost.streetcomplete.osm.surface.SingleSurfaceWithNote
+import de.westnordost.streetcomplete.osm.surface.Surface
+import de.westnordost.streetcomplete.osm.surface.SurfaceAnswer
+import de.westnordost.streetcomplete.osm.surface.SurfaceInfo
+import de.westnordost.streetcomplete.osm.surface.SurfaceMissing
+import de.westnordost.streetcomplete.osm.surface.SurfaceMissingWithNote
+import de.westnordost.streetcomplete.osm.surface.applyTo
+import de.westnordost.streetcomplete.osm.surface.asItem
+import de.westnordost.streetcomplete.osm.surface.commonSurfaceObject
+import de.westnordost.streetcomplete.osm.surface.createSurfaceStatus
+import de.westnordost.streetcomplete.osm.surface.shouldBeDescribed
+import de.westnordost.streetcomplete.osm.surface.toItems
+import de.westnordost.streetcomplete.overlays.AbstractOverlayForm
+import de.westnordost.streetcomplete.overlays.AnswerItem
+import de.westnordost.streetcomplete.overlays.IAnswerItem
+import de.westnordost.streetcomplete.quests.surface.DescribeGenericSurfaceDialog
+import de.westnordost.streetcomplete.util.ktx.nonBlankTextOrNull
+import de.westnordost.streetcomplete.view.image_select.DisplayItem
+import de.westnordost.streetcomplete.view.image_select.ImageListPickerDialog
+import de.westnordost.streetcomplete.view.image_select.ItemViewHolder
+
+class PathSurfaceOverlayForm : AbstractOverlayForm() {
+    override val contentLayoutResId = R.layout.fragment_overlay_path_surface_select
+    private val binding by contentViewBinding(FragmentOverlayPathSurfaceSelectBinding::bind)
+
+    private val itemsPerRow = 2
+    /** items to display. May not be accessed before onCreate */
+    val items: List<DisplayItem<Surface>> = (COMMON_SPECIFIC_PAVED_SURFACES + COMMON_SPECIFIC_UNPAVED_SURFACES + GROUND_SURFACES + GENERIC_AREA_SURFACES).toItems()
+    private val cellLayoutId: Int = R.layout.cell_icon_select_with_label_below
+    private var originalSurfaceStatus: SurfaceInfo? = null
+    private var isSegregatedLayout = false
+
+    private var selectedStatusForMainSurface: DisplayItem<Surface>? = null
+        set(value) {
+            field = value
+            updateSelectedCell()
+        }
+    private var selectedStatusForCyclewaySurface: DisplayItem<Surface>? = null
+        set(value) {
+            field = value
+            updateSelectedCell()
+        }
+    private var selectedStatusForFootwaySurface: DisplayItem<Surface>? = null
+        set(value) {
+            field = value
+            updateSelectedCell()
+        }
+
+        override val otherAnswers: List<IAnswerItem> get() {
+            return if (isSegregatedLayout) {
+                listOf() // removing info about separate cycleway is to complicated
+            } else if (bothFootAndBicycleTraffic(element!!.tags)) {
+                    listOf(
+                        AnswerItem(R.string.overlay_path_surface_segregated) {
```

Comment:
typically it requires editing not only surface info but also an access info. As it happens in cases where bicycle access is gone. It may require also removal of cycleway=separate, bicycle=use_sidepath from the road (yes, adding may also require adding them - but lack of this info on road is much smaller problem than their wrong presence).

And in cases where there is a segregated cycleway with the same surface as footway then StreetComplete will  anyway ask for cycleway:surface and footway:surface.

Fortunately need for this change are really rare. At least in my area I had need to change only in direction of adding cycleway.